packages/meditrak-server/src/apiV2/requestCountryAccess.js

```diff
@@ -84,7 +76,7 @@ export const requestCountryAccess = async (req, res) => {
   } catch (error) {
     throw new UnauthenticatedError(error.message);
   }
-  const userName = await getUserName(userId, models);
+  const userName = await getUserInfoInString(userId, models);
```

Comment:
```suggestion
  const userInfo = await getUserInfoInString(userId, models);
```
And in other places this variable name is used